src/app/main/register/register.component.ts

```diff
@@ -29,9 +29,14 @@
   }
 
   register(){
+    const HTTP_OPTIONS = {
+      headers: new HttpHeaders({
+        'Content-Type': 'aplication/json',
+      })
+    };
     let b = this.form.value
     console.log(b)
-    this._api.postTypeRequest('register', b).subscribe((res: any) => {
+    this._api.postTypeRequest('register', b ,HTTP_OPTIONS).subscribe((res: any) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ticodacytest/chatApp1/pullRequest?prid=7784558)
```suggestion
    this._api.postTypeRequest("register", b ,HTTP_OPTIONS).subscribe((res: any) => {
```